src/ts/helper/AdHelper.ts

```diff
@@ -43,13 +43,14 @@ export interface SsaiAdInfo {
 export class AdHelper {
   public static mapCsaiAdPosition(
     adBreak: AdBreak,
-    player: PlayerAPI,
+    duration: number,
```

Comment:
Similar to above, please rename the parameter to something like `mainCointentDuration`, otherwise it's quite confusing when someone reads this file/method.

Reply:
Renamed https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/118/commits/a0f340727fe1816302b51f74011dd2ef6d801ab6